decidim-core/app/types/decidim/core/user_entity_input_filter.rb

```diff
@@ -46,6 +46,16 @@ class UserEntityInputFilter < BaseInputFilter
                    op_name.or(op_nick)
                  end
                end
+      argument :exclusion,
+               type: String,
```

Comment:
```suggestion
               type: [ID],
```
As we are using ID, we can specify array types of ID, this allows us to use a more idiomatic Graphql call:
```graphql
users(filter:{excludeIds: [2 10 11]}) {
```
or simply:
```graphql
users(filter:{excludeIds: 2}) {
```